demo-app/build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import java.io.FileInputStream
+import java.util.Properties
+
+plugins {
+    id("otel.android-app-conventions")
+}
+
+val localProperties = Properties()
+localProperties.load(FileInputStream(rootProject.file("local.properties")))
+
+android {
+    namespace = "io.opentelemetry.android.demo"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "io.opentelemetry.android.demo"
+        minSdk = 21
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        all {
+            val accessToken = localProperties["rum.access.token"] as String?
+            resValue("string", "rum_access_token", accessToken ?: "fakebroken")
+        }
+        release {
+            isMinifyEnabled = false
```

Comment:
I think it would make sense to have it enabled for the release build as it could allow us to test some real-world scenarios.